ruby/files_and_serialization/project_event_manager.md

```diff
@@ -618,15 +618,18 @@ Let's look for a solution before we attempt to build a solution.
 
 #### Installing the Google API Client
 
-Ruby comes packaged with the `gem` command. This tool allows you to download libraries by knowing the name of the library you want to install.
+In this section, we’ll use [Bundler](https://bundler.io) to manage and install our gem dependencies. Bundler ensures that the correct versions of gems are installed and works seamlessly with different Ruby versions and project requirements.
 
 ```bash
-$ gem install google-api-client
-Successfully installed google-api-client-0.15.0
-1 gem installed
+$ bundle init  # Initializes a Gemfile if one doesn't exist
```

Comment:
```suggestion
$ bundle init  # Initializes a Gemfile if one doesn't exist
```
Don't prefix commands with `$`. This allows learners to more easily copy/paste the commands into their terminal.

Reply:
Thank you, Josh! $ prefix is removed